TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/TeleOpMovement.kt

```diff
@@ -35,6 +36,12 @@ object TeleOpMovement : Component {
             joyMagnitude *= SLOW_MULTIPLIER
         }
 
+        // if right stick y is moved greater than the value of 0.2
+        // use the movement of the linear slide
+        if (gamepad.right_stick_y > 0.2f || gamepad.right_stick_y < -0.2f) {
```

Comment:
```suggestion
        if (abs(gamepad.right_stick_y) > 0.2f)
```

You'll need to import the `abs` function from `kotlin.math` too.